user_docs/en/userGuide.t2t

```diff
@@ -1009,8 +1009,8 @@ The available logging levels are:
 - Disabled: Apart from a brief startup message, NVDA will not log anything while it runs.
 - Info: NVDA will log basic information such as startup messages and information useful for developers.
 - Debug warning: Warning messages that are not caused by severe errors will be logged.
-- Input/output: Input from keyboard and braille displays, as well as speech and braille output will be logged. If you are concerned about privacy, do not set logging level to this option.
-- Debug: In addition to info, warning, and input/output messages, additional debug messages will be logged. Just like input/output, if you are concerned about privacy, you should not set logging level to this option.
+- Input/output: Input from keyboard and braille displays, as well as speech and braille output will be logged. If you are concerned about privacy, do not set the logging level to this option.

```

Comment:
Again for each of these, please split sentences onto new lines. You should also double check that this gets rendered correctly as part of the list.